### PR TITLE
Add counter cache expiration configuration

### PR DESCRIPTION
**Pull request by iguchi1124**
https://github.com/iguchi1124/counter_cache-rails/pull/7#issuecomment-372127011

### CODE REVIEW

File: lib/counter_cache_rails.rb [hunk 1]
```diff
@@ -1,3 +1,18 @@
 # frozen_string_literal: true
 
+require 'counter_cache_rails/configuration'
 require 'counter_cache_rails/railtie'
+
+module CounterCacheRails
+  class << self
+    def configuration
+      @configuration ||= Configuration.new(
+        expires_in: 1.hour
```
Comment (iguchi1124) [inline lib/counter_cache_rails.rb]: TODO: use Rails application default cache expiration limit

**Comment by iguchi1124** [comment]
@Gargron could you review this?

File: lib/counter_cache_rails.rb (hunk 1 above)
Comment (Gargron) [inline lib/counter_cache_rails.rb]: If this is to be consistent with the way built-in counter_cache works, the cache should never expire. A COUNT(*) query is expensive, and doing it every hour would be bad for performance.

Comment (iguchi1124) [reply lib/counter_cache_rails.rb]: I fixed expires_in option set nil by default https://github.com/iguchi1124/counter_cache-rails/pull/8/commits/30c844179ddf4afab8e196d456ed09cbaba54921

**Comment by iguchi1124** [comment]
I configured default `expires_in` value to nil, then this behavior maybe works with built-in counter cache equally. https://github.com/iguchi1124/counter_cache-rails/pull/8/commits/30c844179ddf4afab8e196d456ed09cbaba54921

**Review by Gargron — APPROVED** [review]

**Pull request merged**